packages/components/filters/filters-list/src/filters-list.tsx

```diff
@@ -1,11 +1,135 @@
+import { type ReactNode, useState } from 'react';
+import { css } from '@emotion/react';
+import { MenuProps, MenuListProps } from 'react-select';
+import DropdownMenu from '@commercetools-uikit/dropdown-menu';
+import FilterMenu from '@commercetools-uikit/filter-menu';
+import SecondaryButton from '@commercetools-uikit/secondary-button';
+import SelectInput from '@commercetools-uikit/select-input';
+
+export type TAppliedFilterValue = {
+  filterKey: string;
+  /** does ReactNode | ReactNode[] make sense for single/multiple labels (shown in FilterMenu Chips) */
+  label: ReactNode[];
+};
+
+export type TFiltersConfiguration = {
+  key: string;
+  groupKey?: string;
+  label: ReactNode;
+  filter: ReactNode;
+  isPersistent?: boolean;
+  isDisabled?: boolean;
+  onClearRequest: Function;
+  onApplyRequest?: Function;
+  onSortRequest?: Function;
+};
+
+export type TFilterGroup = {
+  key: string;
+  label: ReactNode;
+};
+
 export type TFiltersListProps = {
-  /**
-   * This is a stub prop
-   */
-  label?: string;
+  filters: TFiltersConfiguration[];
+  filterGroups?: TFilterGroup[];
+  appliedFilters: TAppliedFilterValue[];
+  onClearAllRequest: Function;
+  onAddFilterRequest?: Function;
 };
 
+const CustomSelectMenu = ({
+  children,
+  innerProps: { ref, ...restInnerProps },
+}: MenuProps) => (
+  <div ref={ref} {...restInnerProps}>
+    {children}
+  </div>
+);
+
+const CustomMenuList = ({
+  children,
+  innerProps: { ref, ...restInnerProps },
+}: MenuListProps) => (
+  <div ref={ref} {...restInnerProps}>
+    {children}
+  </div>
+);
+
 function FiltersList(props: TFiltersListProps) {
-  return <div>{props.label}</div>;
+  const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+
+  const persistedFilters = props.filters
+    .filter((filter) => filter.isPersistent)
+    .map((filter) => filter.key);
+
+  const getFilterOptions = () => {
+    if (props.filterGroups) {
+      return props.filterGroups.map((filterGroup) => ({
+        label: filterGroup.label,
+        options: props.filters
+          .filter((filter) => filter.groupKey === filterGroup.key)
+          .map((filter) => ({ value: filter.key, label: filter.label })),
+      }));
+    } else {
+      return props.filters.map((filter) => ({
+        value: filter.key,
+        label: filter.label,
+      }));
+    }
+  };
```

Comment:
Couldn't this also just be part of an initial config if me make a distinction between possible filters and filters that are in use?

Reply:
Are you saying that we would pass in the `options` for the "add filters" menu as a separate config array?